h2o-docs/src/product/flow.rst

```diff
@@ -1327,7 +1327,7 @@ The lower-left side of the graph represents less tolerance for false positives w
 Partial Dependence Plots
 ------------------------
 
-For models that include only numerical values, you can view a Partial Dependence Plot (PDP) for that model. This provides a graphical representation of the marginal effect of a variable on the class probability (classification) or response (regression). 
+For models that include only numerical values, you can view a Partial Dependence Plot (PDP) for that model. This provides a graphical representation of the marginal effect of a variable on the class probability (binary and multiclass classification) or response (regression). 
```

Comment:
Could we also add a note that for multiclass classification a user has to define one target class?